avm/res/compute/virtual-machine/main.bicep

```diff
@@ -330,7 +330,7 @@ var formattedUserAssignedIdentities = reduce(
 var identity = !empty(managedIdentities)
   ? {
       type: (extensionAadJoinConfig.enabled ? true : (managedIdentities.?systemAssigned ?? false))
-        ? (!empty(managedIdentities.?userAssignedResourceIds ?? {}) ? 'SystemAssigned,UserAssigned' : 'SystemAssigned')
+        ? (!empty(managedIdentities.?userAssignedResourceIds ?? {}) ? 'SystemAssigned, UserAssigned' : 'SystemAssigned')
```

Comment:
Why add the additional whitespace? Currently the same code is used multiple times in other modules as well.

Reply:
I think this comes down to slight differences in the modules. Some show warnigs if its not 1:1 what his particular RP expects. @pmalarme, I presume this is why you added the whitespace?

Reply:
Yes it is why